lib/runtime-core/src/module.rs

```diff
@@ -77,6 +79,15 @@ pub struct ModuleInfo {
 
     /// Custom sections.
     pub custom_sections: HashMap<String, Vec<u8>>,
+
+    /// Flag controlling whether or not debug information for use in a debugger
+    /// will be generated.
+    pub generate_debug_info: bool,
+
+    #[cfg(feature = "generate-debug-information")]
+    #[serde(skip)]
+    /// Resource manager of debug information being used by a debugger.
+    pub debug_info_manager: jit_debug::JITCodeDebugInfoManager,
```

Comment:
```suggestion
    pub(crate) debug_info_manager: jit_debug::JITCodeDebugInfoManager,
```

related to my comment above.

Reply:
I was a bit worried about the implications of making the first non-public field, but I suppose it'll be fine